packages/opentelemetry/lib/config/instrumentations.js

```diff
@@ -21,17 +18,14 @@ const IGNORED_REQUEST_PATHS = ['/__gtg', '/__health', '/favicon.ico'];
  * @returns {NodeSDKConfiguration['instrumentations']}
  */
 exports.createInstrumentationConfig = function createInstrumentationConfig() {
-	return [
-		getNodeAutoInstrumentations({
-			'@opentelemetry/instrumentation-http': {
-				ignoreIncomingRequestHook
-			},
-			'@opentelemetry/instrumentation-pino': {
-				enabled: false
-			}
-		}),
-		new RuntimeNodeInstrumentation()
-	];
+	return getNodeAutoInstrumentations({
+		'@opentelemetry/instrumentation-http': {
+			ignoreIncomingRequestHook
+		},
+		'@opentelemetry/instrumentation-pino': {
+			enabled: false
+		}
+	});
```

Comment:
The `NodeSDKConfiguration['instrumentations']` type allows either an array of arrays of instrumentations (what we were doing) or an array of instrumentations. This means we can safely just switch to returning the result of the function call instead of wrapping in an unnecessary array.